README.md

```diff
@@ -19,7 +19,20 @@ I encrypt the `keyring-name : password` pair with GnuPG and save it as `secret-f
 
 > I recommend you to **configure Yubikey as GPG smartcard**. The system would just ask you to unlock gnome-keyring with your default GPG software. You may generate a new GPG key for yubikey, or move your existing GPG key into yubikey. Refer to google for these knowledge. 
 
-First, build the project from source.
+First, install packages.
```

Comment:
Not all users are using ubuntu. Please Add a new section named `dependency`, and add an entry `ubuntu 20.04`. Archlinux/Gentoo/fedora/opensuse users may use different packages.